app/src/main/java/org/greenstand/android/TreeTracker/activities/ImageCaptureActivity.kt

```diff
@@ -71,6 +78,7 @@ class ImageCaptureActivity : AppCompatActivity() {
         // Build the image capture use case and attach button click listener
         val imageCapture = ImageCapture(imageCaptureConfig)
         imageCaptureButton.setOnClickListener {
+            deviceOrientation.takeSnapshotAndDisable()
```

Comment:
This call ensures the rotation matrix snapshot is created at the time of the camera button is clicked. Given the rate at which the sensors are generating events, anything else (e.g. at the time of saving the tree) wouldn't accurately present the device orientation at the time of taking the tree's picture.